dist/common/scripts/scylla_coredump_setup

```diff
@@ -62,13 +65,35 @@ ExternalSizeMax=1024G
         with open('/etc/systemd/coredump.conf', 'w') as f:
             conf = f.write(conf_data)
         if args.dump_to_raiddir:
-            rmtree('/var/lib/systemd/coredump')
+            with open('/etc/fstab') as f:
+                fstab = f.readlines()
+            for f in fstab:
+                if re.match(r'^/var/lib/scylla/coredump', f):
+                    print('/etc/fstab already has bind-mount entry', file=sys.stderr)
+                    sys.exit(1)
             makedirs('/var/lib/scylla/coredump')
-            os.symlink('/var/lib/scylla/coredump', '/var/lib/systemd/coredump')
+            with open('/etc/fstab', 'a') as f:
+                f.write('/var/lib/scylla/coredump /var/lib/systemd/coredump none defaults,bind 0 0\n')
+            run('mount /var/lib/systemd/coredump')
         run('systemctl daemon-reload')
         if is_debian_variant():
             run('sysctl -p /usr/lib/sysctl.d/50-coredump.conf')
         else:
             with open('/etc/sysctl.d/99-scylla-coredump.conf', 'w') as f:
                 f.write('kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e"')
             run('sysctl -p /etc/sysctl.d/99-scylla-coredump.conf')
+
+        fp = tempfile.NamedTemporaryFile()
+        fp.write(b'kill -11 $$')
```

Comment:
kill --SEGV is more portable, some archs have different signal numbers.

Reply:
Or maybe python's os.kill() is even simpler.

Reply:
Updated the code using "-SEGV".